suite-native/app/e2e/utils.ts

```diff
@@ -0,0 +1,57 @@
+import { resolveConfig } from 'detox/internals';
+
+const APP_LAUNCH_ARGS = {
+    // Do not synchronize communication with the trezor bridge and metro server running on localhost. Since the trezor
+    // bridge is exchanging messages with the app all the time, the test runner would wait forever otherwise.
+    detoxURLBlacklistRegex: '\\("^.*127.0.0.1.*",".*localhost.*"\\)',
+};
+
+const platform = device.getPlatform();
+
+export const sleep = (ms: number) => new Promise(res => setTimeout(res, ms));
+
+const getExpoDeepLinkUrl = () => {
+    const expoLauncherUrl = encodeURIComponent(
+        `http://localhost:8081/index.bundle?platform=${platform}&dev=true&minify=false&disableOnboarding=1`,
```

Comment:
low prio nit: Would be nicer if this function accepted `platform` as parameter and would be defined in `openExpoDevClientApp` where it's used, so these functions don't have to touch outer scope to get this value.